crates/orderbook/src/dto/order.rs

```diff
@@ -43,6 +43,15 @@ pub struct Order {
     pub signature: Signature,
 }
 
+#[serde_as]
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Quote {
+    pub sell_amount: U256,
```

Comment:
Should use `#[serde_as(as = "HexOrDecimalU256")]`.

Reply:
Thanks, added